examples/metrics/non_stateful.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2019, OpenTelemetry Authors
```

Comment:
Technically we are in 2020 now so it would be good to reflect that

Reply:
I believe that should be in a separate PR to address this since all other files say 2019.